view/propertyEditor.js

```diff
@@ -245,77 +244,118 @@ ACS.propertyEditor = function (modelList, // ACS.modelList
 				cell.appendChild(dropdownList);
 			}
 
-			//generate checkbox field for boolean
-			if (comboValues === '' && typetemp === ACS.dataType.BOOLEAN) {
+			if (isDynProp) {
+
+				log.debug('downloading (dynamic) properties list');
+				setBaseURI(ACS.areBaseURI + '/rest/');
```

Comment:
why this is needed here? shouldn't this be done once somewhere at setup?

Reply:
In the C# ACS the dyn properties are only fetched, if the ACS is connected to the ARE and the currently edited model is also deployed to the ARE.
-->
You should only fetch the dyn properties, if the ARE Status is
![image](https://user-images.githubusercontent.com/4621810/54362985-47683480-466a-11e9-8d5e-9fec08422526.png)
In all other cases please show a normal text input field filled out with the current value of the field.

Reply:
Dynamic properties are now loading only when ARE and WebACS are synchronised. 

In all other cases, only an input field is displayed for dynamic properties (with the current value, see https://github.com/asterics/WebACS/pull/36/commits/e4592daca1b0c20471cccd3c5302e91ab763e84a)